core/src/main/java/org/apache/accumulo/core/metadata/TabletFile.java

```diff
@@ -104,21 +110,50 @@ public String getFileName() {
   }
 
   /**
-   * Exact string that is stored in the metadata table
+   * Return a string for reading the tablet file. Doesn't have to be exact string in metadata.
    */
-  public String getMetadataEntry() {
-    return metadataEntry;
+  public String getMetaRead() {
```

Comment:
The javadoc comment say its does not have to be the exact path in meta, but the method name has Meta in it.  I feel like Meta should be dropped from the name.